src/main/java/io/ncbpfluffybear/supserv/SupServItemSetup.java

```diff
@@ -88,45 +88,95 @@ public static void setup(@Nonnull SupServPlugin plugin) {
+                new SlimefunItemStack(SupServItems.COMPRESSED_COBBLESTONE, 1)
+        );
+
+        ((MultiBlockMachine) SlimefunItems.COMPRESSOR.getItem()).addRecipe(new ItemStack[] {
+                new SlimefunItemStack(SupServItems.COMPRESSED_COBBLESTONE, 9)},
+                new SlimefunItemStack(SupServItems.DOUBLE_COMPRESSED_COBBLESTONE, 1)
+        );
+
+        ((MultiBlockMachine) SlimefunItems.COMPRESSOR.getItem()).addRecipe(new ItemStack[] {
+                new SlimefunItemStack(SupServItems.DOUBLE_COMPRESSED_COBBLESTONE, 9)},
+                new SlimefunItemStack(SupServItems.TRIPLE_COMPRESSED_COBBLESTONE, 1)
+        );
+
+        ((MultiBlockMachine) SlimefunItems.COMPRESSOR.getItem()).addRecipe(new ItemStack[] {
+                new SlimefunItemStack(SupServItems.TRIPLE_COMPRESSED_COBBLESTONE, 9)},
+                new SlimefunItemStack(SupServItems.QUADRUPLE_COMPRESSED_COBBLESTONE, 1)
+        );
+
+        ((MultiBlockMachine) SlimefunItems.COMPRESSOR.getItem()).addRecipe(new ItemStack[] {
+                new SlimefunItemStack(SupServItems.QUADRUPLE_COMPRESSED_COBBLESTONE, 9)},
+                new SlimefunItemStack(SupServItems.QUINTUPLE_COMPRESSED_COBBLESTONE, 1)
+        );
+
+        ((MultiBlockMachine) SlimefunItems.COMPRESSOR.getItem()).addRecipe(new ItemStack[] {
+                new SlimefunItemStack(SupServItems.QUINTUPLE_COMPRESSED_COBBLESTONE, 9)},
+                new SlimefunItemStack(SupServItems.SEXTUPLE_COMPRESSED_COBBLESTONE, 1)
+        );
+
+        ((MultiBlockMachine) SlimefunItems.COMPRESSOR.getItem()).addRecipe(new ItemStack[] {
+                new SlimefunItemStack(SupServItems.SEXTUPLE_COMPRESSED_COBBLESTONE, 9)},
+                new SlimefunItemStack(SupServItems.SEPTUPLE_COMPRESSED_COBBLESTONE, 1)
+        );
+
+        ((MultiBlockMachine) SlimefunItems.COMPRESSOR.getItem()).addRecipe(new ItemStack[] {
+                new SlimefunItemStack(SupServItems.SEPTUPLE_COMPRESSED_COBBLESTONE, 9)},
+                new SlimefunItemStack(SupServItems.QUADRUPLE_COMPRESSED_COBBLESTONE, 1)
+        );
+
+        // Grind Stone
+        new SlimefunItem(SupServItems.SUPSERV_CATEGORY, SupServItems.COBBLESTONE,
+                RecipeType.GRIND_STONE,
+                new ItemStack[] {SupServItems.COMPRESSED_COBBLESTONE},
+                new ItemStack(Material.COBBLESTONE, 9)
+        ).register(plugin);
+
         new SlimefunItem(SupServItems.SUPSERV_CATEGORY, SupServItems.COMPRESSED_COBBLESTONE,
-            RecipeType.COMPRESSOR, new ItemStack[] {
-                    new ItemStack(Material.COBBLESTONE, 9)
-        }).register(plugin);
+                RecipeType.GRIND_STONE,
+                new ItemStack[] {SupServItems.DOUBLE_COMPRESSED_COBBLESTONE},
+                new SlimefunItemStack(SupServItems.COMPRESSED_COBBLESTONE, 9)
+        ).register(plugin);
 
         new SlimefunItem(SupServItems.SUPSERV_CATEGORY, SupServItems.DOUBLE_COMPRESSED_COBBLESTONE,
```

Comment:
Not sure if this is because I'm looking at it from a phone, but these should not be touching the left side, indent it once so it has the same indentation as the closing parentheses.

Reply:
They look correct on my end unless I am misunderstanding